src/policy/rbf.cpp

```diff
@@ -2,9 +2,13 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <consensus/validation.h>
+#include <logging.h>
 #include <policy/rbf.h>
 #include <util/rbf.h>
 
+#include <string>
```

Comment:
This header doesn't seem to be directly needed in this compilation unit (strings _are_ used as return values/parameters for `uint256.ToString()` and `strprintf`, but it's the job of the headers offering those functions to already also include `<string>`).
```suggestion
```

Reply:
Done